tests/admin_views/tests.py

```diff
@@ -864,6 +864,27 @@ def test_adminsite_display_site_url(self):
         self.assertEqual(response.context['site_url'], '/my-site-url/')
         self.assertContains(response, '<a href="/my-site-url/">View site</a>')
 
+    def test_submit_row_templatetag(self):
+        """
+        #13875 - submit_row templatetag should pass whole context
+        """
+        from django.contrib.admin.templatetags.admin_modify import submit_row
```

Comment:
are these imports ok here? I took a look around and I've noticed other tests perform imports in the method definition if those modules are used only in those tests